elfpy/agents/policies/lp_and_withdraw.py

```diff
@@ -54,3 +57,52 @@ def action(self, market: hyperdrive_market.Market) -> "list[types.Trade]":
                     )
                 )
         return action_list
+
+
+class LpAndWithdrawAgent(agent.AgentFP):
+    """
+    simple LP
+    only has one LP open at a time
+    """
+
+    def __init__(self, wallet_address, budget: FixedPoint = FixedPoint("1000.0")):
+        """call basic policy init then add custom stuff"""
+        self.time_to_withdraw = FixedPoint("1.0")
+        self.amount_to_lp = FixedPoint("100.0")
+        super().__init__(wallet_address, budget)
+
+    def action(self, market: hyperdrive_market.MarketFP) -> List[types.Trade]:
```

Comment:
`List` with a capital L is being phased out (as are similar types like `Dict`, `Tuple`, etc). The correct behavior is to add `from __future__ import annotations` and then use lower-case `list`. Then, when we move to 3.11+, we can remove the import and everything will work.